main.py

```diff
@@ -1,7 +1,37 @@
+from django.contrib.contenttypes.fields import GenericRel
+
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    gernes = ["Western", "Action", "Drama"]
```

Comment:
There is a typo in the variable name `gernes`. It should be `genres` to match the context of the data being handled.